beets/dbcore/queryparse.py

```diff
@@ -34,8 +38,14 @@
 )
 
 
-def parse_query_part(part, query_classes={}, prefixes={},
-                     default_class=query.SubstringQuery):
+# TYPING ERROR
+# not sure what the query type should be
```

Comment:
Apparently no I can't because I have no idea what I was referring to. Seems to have been resolved by other changes or is somewhere in the usage. I'll remove the comments.